tsc/scripts/git_count.py

```diff
@@ -0,0 +1,495 @@
+#!/usr/bin/env python3
+
+import subprocess
+import os
+import pprint
+import json
+import time
+from datetime import datetime
+import argparse
+
+#
+# List of repros
+#
+repros = [
+    "SONiC",          "sonic-host-services",  "sonic-mgmt-framework",    "sonic-quagga",     "sonic-swss",         "sonic-ztp",
+    "sonic-dbsyncd",  "sonic-linux-kernel",   "sonic-platform-common",   "sonic-restapi",    "sonic-swss-common",  "sonic-buildimage",
+    "sonic-fips",     "sonic-mgmt",           "sonic-platform-daemons",  "sonic-sairedis",   "sonic-telemetry",
+    "sonic-frr",      "sonic-mgmt-common",    "sonic-py-swsssdk",        "sonic-snmpagent",  "sonic-utilities"
+]
```

Comment:
Per TSC Election doc, the contribution count should include OCP SAI repo

Reply:
sonic-mgmt (test), SONiC (HLD) carry different score than other code PRs, you may need to process them separately